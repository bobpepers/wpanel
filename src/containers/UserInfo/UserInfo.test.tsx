import { shallow } from 'enzyme';
import { createBrowserHistory } from 'history';
import * as React from 'react';
import {
    Router,
} from 'react-router-dom';
import { UserInfoPage } from './';


const history = createBrowserHistory();
// tslint:disable-next-line:no-any
const mock: any = jest.fn();

const defaultProps = {
    match: {
        params: {
            uid: '',
        },
    },
    history: history,
    location: mock,
};

describe('UserInfo test', () => {
    it('should render', () => {
        const wrapper = shallow(<Router history={history}><UserInfoPage {...defaultProps} /></Router>);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
