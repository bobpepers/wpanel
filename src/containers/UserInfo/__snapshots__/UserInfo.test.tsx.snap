// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo test should render 1`] = `
<Connect(UserInfoScreen)
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/tower",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
  location={[Function]}
  match={
    Object {
      "params": Object {
        "uid": "",
      },
    }
  }
/>
`;
